### PR TITLE
implementing the alarm and Gameselector screen

### PR DESCRIPTION
**Pull request by monashamsolebad**
The alarm was implemented.
The first screen controls the alarm.
When the alarm is played, the game selector page opens and the user should select a game to play to stop the alarm.

### CODE REVIEW

File: src/sample/Controller/AlarmController.java [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+import javafx.scene.media.MediaPlayer;
+
+import java.io.File;
+
+public class AlarmController {
+    private MediaPlayer mediaPlayer;
+
+    public void playAlarm()
+    {
+       // System.err.println(getClass().getResource("/Users/monashamsolebad/sleepyhead/res/alarm.mp3").toString());
```
Comment (ryonryon) [inline src/sample/Controller/AlarmController.java:14]: What is this comment for?

File: src/sample/Controller/FirstScreenController.java [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+import javafx.fxml.Initializable;
+import javafx.scene.Parent;
+import javafx.scene.Scene;
+import javafx.scene.control.Button;
+import javafx.scene.control.Label;
+import javafx.scene.media.Media;
+import javafx.scene.media.MediaPlayer;
+import javafx.stage.Stage;
+
+
```
Comment (ryonryon) [inline src/sample/Controller/FirstScreenController.java:16]: Unnecessary two line break.

File: src/sample/Controller/FirstScreenController.java [hunk 3]
```diff
@@ -0,0 +1,129 @@
...
+import java.io.File;
+import java.net.URL;
+import java.util.ResourceBundle;
+import java.util.Timer;
+import java.util.TimerTask;
+
+// Delegate Pattern
```
Comment (ryonryon) [inline src/sample/Controller/FirstScreenController.java:23]: What is this comments for?

File: src/sample/Controller/GameSelectorController.java [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+
+import javafx.fxml.Initializable;
+
+import java.net.URL;
+import java.util.ResourceBundle;
+
+public class GameSelectorController implements Initializable {
+
+
+
```
Comment (ryonryon) [inline src/sample/Controller/GameSelectorController.java:11]: Unnecessary line break

File: src/sample/Controller/SetAlarmScreenController.java [hunk 5]
```diff
@@ -0,0 +1,122 @@
+package sample.Controller;
+
+
```
Comment (ryonryon) [inline src/sample/Controller/SetAlarmScreenController.java:3]: Unnecessary line break

File: src/sample/Controller/SetAlarmScreenController.java [hunk 6]
```diff
@@ -0,0 +1,122 @@
...
+    @FXML
+    private Button quitButton;
+    private boolean timerValidation = true;
+
+
+    @FXML
+    private void setAlarm(ActionEvent event) {
+
+
```
Comment (ryonryon) [inline src/sample/Controller/SetAlarmScreenController.java:47]: Unnecessary line break

File: src/sample/Controller/SetAlarmScreenController.java [hunk 7]
```diff
@@ -0,0 +1,122 @@
...
+    /**
+     * When you have to initialize your data.
+     *
+     * @param url
+     * @param rb
+     */
+    @Override
+    public void initialize(URL url, ResourceBundle rb) {
+
```
Comment (ryonryon) [inline src/sample/Controller/SetAlarmScreenController.java:113]: Unnecessary line break

File: src/sample/Controller/SetAlarmScreenController.java [hunk 8]
```diff
@@ -0,0 +1,122 @@
...
+
+
+    }
+
+}
+
```
Comment (ryonryon) [inline src/sample/Controller/SetAlarmScreenController.java:119]: Unnecessary line break

**Review by ryonryon — COMMENTED** [review]
@monashamsolebad  @yoniakabecky Can you check my review?

File: src/sample/Main.java [hunk 9]
```diff
@@ -7,31 +7,21 @@
...
 public class Main extends Application {
 
-    private static Stage stage;
-
     @Override
-    public void start(Stage primaryStage) throws Exception{
-        Main.stage = primaryStage;
-        changeView("View/firstScreen.fxml");
-        primaryStage.setTitle("Welcome to the Sleepyhead App");
-        primaryStage.setResizable(false);
```
Comment (yoniakabecky) [inline src/sample/Main.java:22]: I think line 22 is necessary (line 21 is optional)

File: src/sample/Controller/AlarmController.java (hunk 1 above)
Comment (monashamsolebad) [reply src/sample/Controller/AlarmController.java:14]: I used this line but I don't need it anymore

File: src/sample/Controller/FirstScreenController.java (hunk 2 above)
Comment (monashamsolebad) [reply src/sample/Controller/FirstScreenController.java:16]: ok

File: src/sample/Controller/FirstScreenController.java (hunk 3 above)
Comment (monashamsolebad) [reply src/sample/Controller/FirstScreenController.java:23]: just for my own information

File: src/sample/Controller/GameSelectorController.java (hunk 4 above)
Comment (monashamsolebad) [reply src/sample/Controller/GameSelectorController.java:11]: ok

File: src/sample/Controller/SetAlarmScreenController.java (hunk 5 above)
Comment (monashamsolebad) [reply src/sample/Controller/SetAlarmScreenController.java:3]: ok

File: src/sample/Controller/SetAlarmScreenController.java (hunk 6 above)
Comment (monashamsolebad) [reply src/sample/Controller/SetAlarmScreenController.java:47]: not nessaccary

File: src/sample/Controller/SetAlarmScreenController.java (hunk 7 above)
Comment (monashamsolebad) [reply src/sample/Controller/SetAlarmScreenController.java:113]: ok

File: src/sample/Controller/SetAlarmScreenController.java (hunk 8 above)
Comment (monashamsolebad) [reply src/sample/Controller/SetAlarmScreenController.java:119]: ok

File: src/sample/Main.java (hunk 9 above)
Comment (monashamsolebad) [reply src/sample/Main.java:22]: 21 is just for the title of the page. 22 is for that user can't change the size of the window. It is optional too

**Comment by monashamsolebad** [comment]
If you are ok lets merge the pull request

File: src/sample/Main.java (hunk 9 above)
Comment (yoniakabecky) [reply src/sample/Main.java:22]: I know what are 21 and 22 lines do. 
I haven't made the first screen resizable so line 21 is necessary.  If you made the window adjustable it is not necessary.

**Comment by monashamsolebad** [comment]
Sorry I meant that these lines are not necessary for me. If you want to
change them that's fine :)

File: src/sample/Main.java (hunk 9 above)
Comment (yoniakabecky) [reply src/sample/Main.java:22]: Sorry, my bad. I mean 22 is necessary. If you resize the size of first screen, you will understand what I mean. Other things are fine for me. plz merge it :)

**Pull request merged**